### PR TITLE
SAK-40247 Even though a user can edit a poll, they can't see the results.

### PR DESCRIPTION
**Pull request by maramar7**

### CODE REVIEW

File: polls/impl/src/java/org/sakaiproject/poll/service/impl/PollListManagerImpl.java [hunk 1]
```diff
@@ -175,6 +175,11 @@ public boolean savePoll(Poll t) throws SecurityException, IllegalArgumentExcepti
         }
 
         try {
+            if(!newPoll) {
+                Poll poll = dao.findById(Poll.class, Long.parseLong(t.getId()));
+                if(poll != null && !t.getOwner().equals(poll.getOwner()))
```
Comment (bjones86) [inline polls/impl/src/java/org/sakaiproject/poll/service/impl/PollListManagerImpl.java]: Please use braces, even on single line `if/else` statements.

**Comment by bjones86** [comment]
@ern, Codacy has been addressed here, care to re-review?

**Comment by maramar7** [comment]
@bjones86 Done

**Comment by bjones86** [comment]
@maramar7, looks like the tests are still failing:

```
Tests in error: 
  PollListManagerTest.testDeletePoll:198 Â» NumberFormat For input string: "null"
  PollListManagerTest.testSavePoll:123 Â» NumberFormat For input string: "null"

Tests run: 4, Failures: 0, Errors: 2, Skipped: 0
```

**Comment by maramar7** [comment]
@bjones86 Done

**Comment by bjones86** [comment]
There are still some tests failing here...

```
Running org.sakaiproject.poll.logic.test.PollListManagerTest
Tests run: 3, Failures: 1, Errors: 1, Skipped: 0, Time elapsed: 6.234 sec <<< FAILURE! - in org.sakaiproject.poll.logic.test.PollListManagerTest
testSavePoll(org.sakaiproject.poll.logic.test.PollListManagerTest)  Time elapsed: 0.131 sec  <<< FAILURE!
java.lang.AssertionError
	at org.sakaiproject.poll.logic.test.PollListManagerTest.testSavePoll(PollListManagerTest.java:130)

testDeletePoll(org.sakaiproject.poll.logic.test.PollListManagerTest)  Time elapsed: 0.016 sec  <<< ERROR!
java.lang.IllegalArgumentException: Poll id can't be null
	at org.sakaiproject.poll.logic.test.PollListManagerTest.testDeletePoll(PollListManagerTest.java:235)

Running org.sakaiproject.poll.logic.test.PollListManagerImplTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.608 sec - in org.sakaiproject.poll.logic.test.PollListManagerImplTest

Results :

Failed tests: 
  PollListManagerTest.testSavePoll:130
Tests in error: 
  PollListManagerTest.testDeletePoll:235 Â» IllegalArgument Poll id can't be null
```

**Pull request merged**